compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/ClassReference.kt

```diff
@@ -78,6 +79,12 @@ public sealed class ClassReference {
   public abstract fun isAbstract(): Boolean
   public abstract fun visibility(): Visibility
 
+  /**
+   * Returns only the super class (excluding [Any]) and implemented interfaces declared directly by
+   * this class.
+   */
+  public abstract fun directSuperClassReferences(): Sequence<ClassReference>
```

Comment:
That should become a `List`. Sequences are evaluated over and over again and the lazy below becomes meaningless.